app/helpers/application_helper.rb

```diff
@@ -88,14 +88,17 @@ def pretty_json(raw_json_string)
   end
 
   def top_menu_items(format_values=nil)
-    tags = Rails.configuration.x.top_menu_items.collect do |menu_item|
-      (url = menu_item['url'] % format_values) if format_values
+    menu_items = [
+      { 'key' => 'monitoring', 'url' => '/dashboards/:monitoring' },
+      { 'key' => 'tuning', 'url' => '/dashboards/:tuning' }
+    ]
```

Comment:
I don't quite understand what's the point of hard-coding these entries here when we already have all the customizations in `vendor/customizations.json`.
Can't we just add these entries there and leave the helper as it was?

Reply:
reverted to use customization.json